config/owasp/suppressions.xml

```diff
@@ -53,4 +53,11 @@
     <cve>CVE-2021-37137</cve>
     <cve>CVE-2021-43797</cve>
   </suppress>
+  <suppress>
+    <notes><![CDATA[
+   file name: jackson-databind-2.12.6.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/com.fasterxml.jackson.core/jackson-databind@2.12.6</packageUrl>
+    <cve>CVE-2020-36518</cve>
+  </suppress>
```

Comment:
more suppressions?

Reply:
![Screenshot 2022-03-21 at 11 15 19](https://user-images.githubusercontent.com/26488566/159250281-78066519-f6c1-4c95-83b2-5b15cb17ed2e.png)
unfortunately so, checked the root dependency and its the one thats giving us the netty issues too, tried updating to lastest and still persists so leaving it for when we tackle the OWASP issue properly